SPECS/cassandra-cpp-driver/cassandra-cpp-driver.spec

```diff
@@ -0,0 +1,61 @@
+Summary:        DataStax C/C++ Driver for Apache Cassandra and DataStax Products
+Name:           cassandra-cpp-driver
+Version:        2.16.0
+Release:        1%{?dist}
+Epoch:          1
+License:        Apache 2.0
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+Group:          Development/Tools
+URL:            https://github.com/datastax/cpp-driver
+Source0:        https://github.com/datastax/cpp-driver/archive/refs/tags/%{version}.tar.gz#/%{name}-%{version}.tar.gz
+BuildRequires:  cmake
+BuildRequires:  libuv-devel
+BuildRequires:  openssl-devel
+Requires:       libuv
+
+%description
+A modern, feature-rich, and highly tunable C/C++ client library for Apache
+Cassandra and DataStax Products using Cassandra's native protocol and Cassandra
+Query Language along with extensions for DataStax Products.
+
+%package devel
+Summary:        Development libraries for ${name}
+Group:          Development/Tools
+Requires:       %{name} = %{epoch}:%{version}-%{release}
+Requires:       libuv-devel
+
+%description devel
+Development libraries for %{name}
+
+%prep
+%setup -q -n cpp-driver-%{version}
+
+%build
+mkdir -p build
+cd build
+%cmake ..\
+    -DCMAKE_BUILD_TYPE=RELEASE
+%make_build
+
+%install
+cd build
+%make_install
+
+%post -p /sbin/ldconfig
+%postun -p /sbin/ldconfig
```

Comment:
Nit: These two lines can be condensed into one using `%ldconfig_scriptlets`

Reply:
Fixed